File: scratch/check_columns.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

load_dotenv()

url = os.environ.get("SUPABASE_URL")
key = os.environ.get("SUPABASE_SERVICE_ROLE_KEY")
supabase: Client = create_client(url, key)

try:
    res = supabase.table('products').select('*').limit(1).execute()
    if res.data:
        print(res.data[0].keys())
    else:
        print("No data in products table")
except Exception as e:
    print(e)

File: scratch/check_columns.js
const { createClient } = require('@supabase/supabase-js');
require('dotenv').config();

const url = process.env.SUPABASE_URL;
const key = process.env.SUPABASE_ANON_KEY;
const supabase = createClient(url, key);

async function check() {
    const { data, error } = await supabase.from('branch_inventory').select('*').limit(1);
    if (error) {
        console.error(error);
    } else if (data && data.length > 0) {
        console.log(Object.keys(data[0]));
    } else {
        console.log("No data");
    }
}

check();
